lib/setup.js

```diff
@@ -36,6 +45,12 @@ var constructResponse = function (server) {
           path: route.path,
           plugin: route.public.realm.plugin || null
         };
+      }).sort(function (a, b) {
+        var pathCompare = a.path.localeCompare(b.path);
+        if (pathCompare === 0) {
+          return methodSort[a.method] - methodSort[b.method];
```

Comment:
Should probably uppercase the methods, just in case. Also, what happens if a method sneaks in besides the ones we check for?